src/modules/auth/auth.service.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  BadRequestException,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { LoginDto } from './dto/login.dto';
+import * as bcryptjs from 'bcryptjs';
+import { JwtService } from '@nestjs/jwt';
+import { ResestPasswordDto } from './dto/resetPassword.dto';
+import { UsersService } from '../users/users.service';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    private readonly jwtService: JwtService,
+    private readonly usersSevice: UsersService,
```

Comment:
awesome! this is helpful to inject the services!

Reply:
Got it thank u for all the feedback